src/components/ConfigEditor/ConfigEditor.tsx

```diff
@@ -1,17 +1,18 @@
 import { AwsAuthDataSourceJsonData, AwsAuthDataSourceSecureJsonData, ConnectionConfig } from '@grafana/aws-sdk';
 import { DataSourcePluginOptionsEditorProps } from '@grafana/data';
-import React, { PureComponent } from 'react';
+import React from 'react';
 import { standardRegions } from './regions';
+import { config } from '@grafana/runtime';
 
 export type Props = DataSourcePluginOptionsEditorProps<AwsAuthDataSourceJsonData, AwsAuthDataSourceSecureJsonData>;
 
-export class ConfigEditor extends PureComponent<Props> {
-  render() {
-    return (
-      <div>
-        <ConnectionConfig {...this.props} standardRegions={standardRegions} />
-        {/* can add x-ray specific things here */}
-      </div>
-    );
-  }
+export function ConfigEditor(props: Props) {
+  const newFormStylingEnabled = config.featureToggles.awsDatasourcesNewFormStyling;
+
+  return (
+    <div className="width-30">
```

Comment:
just curious if this div is still necessary with the new components

Reply:
It is, since the new components adjust to the width of the container which we have to define at the top here. I don't think we have a fixed width requirement for our datasources (as long as it's not more that [578px,](https://docs.google.com/document/d/11XMaYHSMSra8AoN2-yVW-5QPbjEXgkAFbTIF6N84cGE/edit#heading=h.5querbxloix5) but that looks way too wide)